usr/src/test/libc-tests/runfiles/default.run

```diff
@@ -102,12 +102,10 @@ timeout = 600
 [/opt/libc-tests/tests/call_once.32]
 [/opt/libc-tests/tests/call_once.64]
 [/opt/libc-tests/tests/catopen]
-[/opt/libc-tests/tests/env-OS-4089.32]
-[/opt/libc-tests/tests/env-OS-4089.64]
-[/opt/libc-tests/tests/env-7076.32]
-[/opt/libc-tests/tests/env-7076.64]
 [/opt/libc-tests/tests/endian.32]
 [/opt/libc-tests/tests/endian.64]
+[/opt/libc-tests/tests/env-OS-4089.32]
```

Comment:
This file needs copyright update

Reply:
Because this was a mismerge, I didn't think it did.

Reply:
This file only changed due to earlier mismerge, no copyright change necessary.